scripts/pic32pinctrl.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2024 Microchip
```

Comment:
You should keep my copyrights on **<files>.py** inside the script folder since the content was fully based on the work on the **hal_atmel** developed by me.

```diff
+# Copyright (c) 2022 Gerson Fernando Budke <nandojve@gmail.com>
# Copyright (c) 2024 Microchip
```